recipe/build.sh

```diff
@@ -7,7 +7,6 @@ mkdir -p ${PREFIX}/bin
 mkdir build
 
 pushd build;
-PYTHON_MODULE_EXTENSION=$(python -c "import sysconfig; print(sysconfig.get_config_var('EXT_SUFFIX'))")
```

Comment:
This actually looked promising? Failure was that we don't have `tsl_robin_map` on the aarches

Reply:
trying to build it locally, it seemed like it was downloading tsl_robin_map itself.

Reply:
see "correct" fix.

Reply:
Well, we cannot pull in what we don't have

Reply:
https://github.com/conda-forge/tsl_robin_map-feedstock/pull/9